packages/webapp/src/main/components/modals/share-modal/share-modal.tsx

```diff
@@ -12,6 +12,8 @@ import { ModalContentProps } from '../application-modal-types';
 import { LocalStorageRepository } from '../../../services/local-storage/local-storage-repository';
 import { toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
+// @ts-ignore
+import TooltipIcon from '../../../../../assets/tooltip.svg';
```

Comment:
Please never use `@ts-ignore` in production code please :) 
And i guess you can define some kind of base folder to avoid such long relative paths.